docs_outdoornav_user_manual/features/navigation.mdx

```diff
@@ -6,6 +6,13 @@ toc_min_heading_level: 2
 toc_max_heading_level: 4
 ---
 
+:::danger SAFETY WARNING
+
+Making changes to any of the following variables may cause system safety concerns. It is recommended that users
+only modify these parameters in consultation with CPR.

```

Comment:
Do we use "CPR" elsewhere in the docs? Or should it be "Clearpath Robotics Support"?